flink-libraries/flink-table/src/test/java/org/apache/flink/table/runtime/utils/JavaUserDefinedAggFunctions.java

```diff
@@ -86,6 +86,13 @@ public Long getValue(WeightedAvgAccum accumulator) {
 		}
 
 		// overloaded accumulate method
+		// dummy to test constants
+		public void accumulate(WeightedAvgAccum accumulator, long iValue, int iWeight, int x, String string) {
+			accumulator.sum += iWeight + Integer.parseInt(string);
```

Comment:
change the method to 
```
accumulator.sum += (iValue + Integer.parseInt(string)) * iWeight;
accumulator.count += iWeight;
``` 

to have some influence of the value of `string` in the result?